priv/repo/migrations/20230616114036_add_selected_checks_to_hosts.exs

```diff
@@ -0,0 +1,9 @@
+defmodule Trento.Repo.Migrations.AddSelectedChecksToHosts do
+  use Ecto.Migration
+
+  def change do
+    alter table(:hosts) do
+      add(:selected_checks, {:array, :string}, default: [])
```

Comment:
Remove the parenthesis please, added incorrectly by the formatter.
`add :selected_checks, {:array, :string}, default: []`
Check other files

Reply:
Thanks i had to disable the vs code extension xD